packages/types/src/options.ts

```diff
@@ -153,16 +153,11 @@ export interface ClientOptions<TO extends BaseTransportOptions = BaseTransportOp
   tunnel?: string;
 
   /**
-   * Important: This option is currently unused and will only work in the next major version of the SDK
```

Comment:
Rather than just deleting this line, I think we should say that it works client-side in the nextjs SDK but not elsewhere.